front/src/__tests__/organisms/SettingsManager.test.tsx

```diff
@@ -1,33 +1,144 @@
+import { vi } from 'vitest'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { render, screen, waitFor, fireEvent } from '../test-utils'
+import { TAuthContext, useAuth } from '../../context/AuthProvider'
 import { SettingsManager } from '../../components/organisms'
 
+const mockUsers = [
+  {
+    id: '1',
+    email: 'user1@example.com',
+    dni: '12345678',
+    name: 'User One',
+    status: 'ACTIVE',
+    role: 'user',
+    createdAt: '2023-01-01T00:00:00Z',
+    updatedAt: '2023-01-01T00:00:00Z',
+  },
+]
+vi.mock('/api/users', () => mockUsers)
+const queryClient = new QueryClient()
+
+const renderWithQueryClient = (component: React.ReactNode) =>
+  render(
+    <QueryClientProvider client={queryClient}>{component}</QueryClientProvider>
+  )
+
+beforeEach(() => {
+  vi.mock('../../context/AuthProvider', async () => {
+    const actual: Record<number, unknown> = await vi.importActual(
+      '../../context/AuthProvider'
+    )
+    return {
+      ...actual,
+      useAuth: vi.fn(),
+    }
+  })
+  vi.mocked(useAuth).mockReturnValue({
+    user: {
+      name: 'MentorName',
+      avatar: 'MentorAvatar',
+      role: 'MENTOR',
+    },
+  } as TAuthContext)
+})
+
+afterEach(() => {
+  vi.resetAllMocks()
+})
+
 describe('SettingsManager component', () => {
-  it('renders correctly', () => {
+  it('renders Temes and Usuaris tabs correctly for mentor roles or higher', () => {
     render(<SettingsManager />)
 
     expect(screen.getByText('Temes')).toBeInTheDocument()
-    expect(screen.getByText(/usuaris/i)).toBeInTheDocument()
-    expect(
-      screen.getByText(
-        /No hi ha temes disponibles. *Accedeix des d'una categoria per veure o gestionar els temes./
-      )
-    ).toBeInTheDocument()
```

Comment:
This text is still available, isn't it? 
Or what happens in the Temes tab if a Mentor/the Admin opens the SettingsManager from Home page, where there is not slug (needed to render topics)?